pre-docs/bolt_installing.md

```diff
@@ -209,4 +209,10 @@ To disable the collection of analytics data add the following line to `~/.puppe
 ```
 disabled: true
 ```
+## A Note on Kerberos
+
+We support using [Kerberos]() for authentication, but due to license incompatibility with other components we are distributing you'll need to install the gem yourself. You can do this with
```

Comment:
Why is `Kerberos` a link?

Reply:
Also, this needs to be more specific: `We support using Kerberos over SSH for authentication,`.